unit_test/pow.cpp

```diff
@@ -26,23 +26,24 @@ BOOST_AUTO_TEST_CASE(util_parses_a_valid_ttl) {
 }
 
 BOOST_AUTO_TEST_CASE(it_checks_a_valid_pow) {
-    const auto nonce = "AAAAAAABBCE=";
-    const auto timestamp = "1549252654";
+    const auto nonce = "AAAAAAAaFio=";
```

Comment:
Could we define these as constants and reuse them for all the tests, just varying the specific field that is being modified?